std.go

```
@@ -112,21 +269,63 @@ func Reset() {
-func GetEvent(name string) (Event, bool) {
+func GetEvent(name string) (Event[any], bool) {
 	return std.GetEvent(name)
 }
 
 // HasEvent has event check.
 func HasEvent(name string) bool {
 	return std.HasEvent(name)
 }
+
+// subscriberTToAnyAdapter adapts a Subscriber[T] to a Subscriber[any]
+type subscriberTToAnyAdapter[T any] struct {
```

golangci-lint (1.64.8)

type `subscriberTToAnyAdapter` is unused

(unused)